apps/prairielearn/src/api/v1/error.ts

```diff
@@ -1,12 +1,12 @@
-// @ts-check
+import type { ErrorRequestHandler } from 'express';
 import status from 'http-status';
 
 import { logger } from '@prairielearn/logger';
 
-export default function (err, req, res, _next) {
+export default (function (err, req, res, _next) {
   logger.error('API Error', err);
   const statusCode = err.status || 500;
   res.status(statusCode).send({
     message: status[statusCode],
   });
-}
+} satisfies ErrorRequestHandler as ErrorRequestHandler);
```

Comment:
Is there a reason you used both `satisfies` and `as`? AFAICT it works with just `satisfies`.

```suggestion
} satisfies ErrorRequestHandler);
```

Reply:
https://github.com/microsoft/TypeScript/issues/13626#issuecomment-1472348545

Reply:
I guess to get more specific: why does it matter that the type is in fact `ErrorRequestHandler` and not whatever TS infers? That could matter if you're dealing with more complex objects, but this is simply a function. Since the compiler doesn't complain if `as ErrorRequestHandler` isn't present, it seems strictly unnecessary to me.

Reply:
Good point. Should I remove it from the error page as well?

https://github.com/PrairieLearn/PrairieLearn/blob/499f825e98748821c76d449658a4e285ba1f9930/apps/prairielearn/src/pages/error/error.ts#L52

Reply:
Go for it!